content/references/http-websocket-apis/public-api-methods/server-info-methods/server_info-clio.md

```diff
@@ -253,47 +676,52 @@ The `info` object may have some arrangement of the following fields:
-| `time`                              | String          | The current time in UTC, according to the server's clock. [Updated in: rippled 1.5.0][] |
-| `uptime`                            | Number          | Number of consecutive seconds that the server has been operational. [New in: rippled 0.30.1][] |
+| `clio_version`                      | String          |  The version number of the running `Clio` server.  |
+| `validation_quorum`                 | Number          | Minimum number of trusted validations required to validate a ledger version. Some circumstances may cause the server to require more validations. |
+| `rippled_version`                   | String          |  The version number of the running `rippled` server that the `Clio` server is connected to. |
 | `validated_ledger`                  | Object          | _(May be omitted)_ Information about the most recent fully-validated ledger. If the most recent validated ledger is not available, the response omits this field and includes `closed_ledger` instead. |
 | `validated_ledger.age`              | Number          | The time since the ledger was closed, in seconds. |
 | `validated_ledger.base_fee_xrp`     | Number          | Base fee, in XRP. This may be represented in scientific notation such as `1e-05` for 0.00001. |
 | `validated_ledger.hash`             | String          | Unique hash for the ledger, as hexadecimal. |
 | `validated_ledger.reserve_base_xrp` | Number          | Minimum amount of XRP (not drops) necessary for every account to keep in reserve |
 | `validated_ledger.reserve_inc_xrp`  | Number          | Amount of XRP (not drops) added to the account reserve for each object an account owns in the ledger. |
 | `validated_ledger.seq`              | Number          | The [ledger index][] of the latest validated ledger. |
-| `validation_quorum`                 | Number          | Minimum number of trusted validations required to validate a ledger version. Some circumstances may cause the server to require more validations. |
 | `validator_list_expires`            | String          | _(Admin only)_ Either the human readable time, in UTC, when the current validator list will expire, the string `unknown` if the server has yet to load a published validator list or the string `never` if the server uses a static validator list. [Updated in: rippled 1.5.0][] |
+| `cache`                             | Object          |  Data retrieved from the `rippled` server is stored in efficiently in the `cache` object. Thus resulting in a higher throughput and sometimes lower latency for Clio API requests. |
+| `cache.size`                        | Number          |  Size of the `cache` object containing the validated ledger data retrieved from P2P `rippled` server. |
+| `cache.is_full`                     | Boolean         | This is true if the cache is full or not empty. |
```

Comment:
True if cache contains all state data for a specific ledger, false otherwise. Some RPCs, such as book_offers, process much faster when the cache is full.